### PR TITLE
fixing single quotations and apostrophs

### PR DESCRIPTION
**Pull request by entorb**

### CODE REVIEW

File: chapters/hpmor-chapter-000.tex [hunk 1]
```diff
@@ -17,10 +17,10 @@ \section*{Something, somewhere, somewhen, must have happened differently…}
 
 \begin{itemize}
 \item \textsc{Petunia Evans} married Michael Verres, a Professor of Biochemistry at Oxford.
-\item \textsc{Harry James Potter-Evans-Verres} grew up in a house filled to the brim with books. He once bit a math teacher who didn't know what a logarithm was. He's read \emph{Gödel, Escher, Bach} and \emph{Judgment Under Uncertainty: Heuristics and Biases} and volume one of \emph{The Feynman Lectures on Physics}. And despite what everyone who's met him seems to fear, he doesn't want to become the next Dark Lord. He was raised better than that. He wants to discover the laws of magic and become a god.
+\item \textsc{Harry James Potter-Evans-Verres} grew up in a house filled to the brim with books. He once bit a math teacher who didn‘t know what a logarithm was. He’s read \emph{Gödel, Escher, Bach} and \emph{Judgment Under Uncertainty: Heuristics and Biases} and volume one of \emph{The Feynman Lectures on Physics}. And despite what everyone who’s met him seems to fear, he doesn’t want to become the next Dark Lord. He was raised better than that. He wants to discover the laws of magic and become a god.
```
Comment (rrthomas) [inline chapters/hpmor-chapter-000.tex:20]: "didn‘t" is wrong.

File: chapters/hpmor-chapter-000.tex [hunk 2]
```diff
@@ -34,9 +34,9 @@ \section*{Something, somewhere, somewhen, must have happened differently…}
 % \end{center}
 
 \section*{Some notes}
-The opinions of characters in this story are not necessarily those of the author. What warm!Harry thinks is \emph{often} meant as a good pattern to follow, especially if Harry thinks about how he can cite scientific studies to back up a particular principle. But not everything Harry does or thinks is a good idea. That wouldn't work as a story. And the less warm characters may sometimes have valuable lessons to offer, but those lessons may also be dangerously double-edged.
+The opinions of characters in this story are not necessarily those of the author. What warm!Harry thinks is \emph{often} meant as a good pattern to follow, especially if Harry thinks about how he can cite scientific studies to back up a particular principle. But not everything Harry does or thinks is a good idea. That wouldn’s work as a story. And the less warm characters may sometimes have valuable lessons to offer, but those lessons may also be dangerously double-edged.
```
Comment (rrthomas) [inline chapters/hpmor-chapter-000.tex:37]: "wouldn’s" is wrong

File: chapters/hpmor-chapter-000.tex [hunk 3]
```diff
@@ -34,9 +34,9 @@ \section*{Something, somewhere, somewhen, must have happened differently…}
...
 
-If you haven't visited \url{https://hpmor.com}, don't forget to do that at some point; otherwise you'll miss out on the fan art, how to learn everything Harry knows, and more.
+If you haven‘t visited \url{https://hpmor.com}, don’t forget to do that at some point; otherwise you'll miss out on the fan art, how to learn everything Harry knows, and more.
```
Comment (rrthomas) [inline chapters/hpmor-chapter-000.tex:39]: "haven‘t" is wrong

File: chapters/hpmor-chapter-012.tex [hunk 4]
```diff
@@ -1,10 +1,10 @@
 \chapter{Impulse Control}
 
 \begin{chapterOpeningAuthorNote}
-ph'nglui mglw'nafh J.~K.~Rowling wgah'nagl fhtagn
+ph‘nglui mglw’nafh J.~K.~Rowling wgah'nagl fhtagn
```
Comment (rrthomas) [inline chapters/hpmor-chapter-012.tex:4]: "ph‘nglui" is wrong

**Review by rrthomas — CHANGES_REQUESTED** [review]
Minor fixes needed!

File: chapters/hpmor-chapter-000.tex (hunk 1 above)
Comment (entorb) [reply chapters/hpmor-chapter-000.tex:20]: fixed, thanks!

File: chapters/hpmor-chapter-008.tex [hunk 5]
```diff
@@ -1,9 +1,9 @@
...
-All these worlds are J.~K.~Rowling's, except Europa. Attempt no fanfics there.
+All these worlds are J.~K.~Rowling’s, except Europa. Attempt no fanfics there.
 
-One alert reviewer asked whether, if Luna is a seer, that means this is going to be an HPDM bottom!Draco mpreg fic. I regret that FFN does not allow me any larger font size in which to say NO. It honestly hadn't occurred to me that Luna might be a \emph{real} seer—I'll have to decide whether to run with that or not—but I think we can all safely assume that if Luna \emph{is} a seer, she said something about “light planting a seed in darkness”, and Xenophilius, as always, interpreted this in rather the wrong way.
+One alert reviewer asked whether, if Luna is a seer, that means this is going to be an HPDM bottom!Draco mpreg fic. I regret that FFN does not allow me any larger font size in which to say NO. It honestly hadn’s occurred to me that Luna might be a \emph{real} seer—I'll have to decide whether to run with that or not—but I think we can all safely assume that if Luna \emph{is} a seer, she said something about “light planting a seed in darkness”, and Xenophilius, as always, interpreted this in rather the wrong way.
```
[the remark this answers is not in the document]
Comment (entorb) [reply chapters/hpmor-chapter-008.tex:6]: fixed, thanks

File: chapters/hpmor-chapter-012.tex (hunk 4 above)
Comment (entorb) [reply chapters/hpmor-chapter-012.tex:4]: reverted

File: chapters/hpmor-chapter-015.tex [hunk 6]
```diff
@@ -3,10 +3,10 @@ \chapter{Conscientiousness}
 \begin{chapterOpeningAuthorNote}
 Love as thou Rowling.
 
-Today's historical tidbit: The ancient Hebrews considered the boundary of a day to be sunset rather than dawn, so they said “evening and morning” not “morning and evening”. (And as many reviewers noted, modern Jewish halacha asserts the same.)
+Today’s historical tidbit: The ancient Hebrews considered the boundary of a day to be sunset rather than dawn, so they said “evening and morning” not “morning and evening”. (And as many reviewers noted, modern Jewish halacha asserts the same.)
 \end{chapterOpeningAuthorNote}
 \begin{chapterOpeningQuote}
-I'm sure I'll find the time somewhere.
+I‘m sure I’ll find the time somewhere.
```
[the remark this answers is not in the document]
Comment (entorb) [reply chapters/hpmor-chapter-015.tex:9]: good catch!

File: chapters/hpmor-chapter-018.tex [hunk 7]
```diff
@@ -4,7 +4,7 @@ \chapter{Dominance Hierarchies}
 Any sufficiently advanced J.~K.~Rowling is indistinguishable from magic.
 \end{chapterOpeningAuthorNote}
 \begin{chapterOpeningQuote}
-That does sound like the sort of thing I would do, doesn't it?
+That does sound like the sort of thing I would do, doesn’s it?
```
[the remark this answers is not in the document]
Comment (entorb) [reply chapters/hpmor-chapter-018.tex:7]: fixed too, sorry for the mess!

**Review by rrthomas — APPROVED** [review]
This looks good! I think all the fixes are apostrophes, so the fix commit can be squashed into the main apostrophes one (also note English spelling "apostrophes" for the commit message).

**Comment by entorb** [comment]
done

**Pull request merged**